### PR TITLE
Add infoMessage prop.

### PR DESCRIPTION
**Pull request by salesfelipe**
#### What does this PR do? \*

- Add **infoMessage** prop to the directive.

#### How to test it? \*

- Clone the repo and the oms one:

_within ng-pagination:_

    grunt oms

_within oms-ui:_

    grunt

### CODE REVIEW

File: src/vtex-ng-filter.html [hunk 1]
```diff
@@ -13,21 +13,24 @@ <h3>
     </button>
   </h3>
 
+  <span ng-if="infoMessage" translate>{{ infoMessage }}</span>
```
Comment (ohmyguigs) [inline src/vtex-ng-filter.html]: nice backward compatibility bro

File: src/vtex-ng-filter.html [hunk 2]
```diff
@@ -53,8 +58,8 @@ <h3 class="group-header" ng-if="group && group.length > 0">
                 type="checkbox"
                 ng-model="filter.useTimezoneOffset"
                 ng-change="filter.onUseTimezoneOffsetChange()"
-                name="filter{{$index}}-use-timezone-offset"
-              > 
+                name="filter{{ $index }}-use-timezone-offset"
+              />
```
Comment (ohmyguigs) [inline src/vtex-ng-filter.html]: maybe lint didn't get this one? (changed with line 125 ? 🤔)

File: src/vtex-ng-filter.html [hunk 3]
```diff
@@ -115,9 +120,9 @@ <h3 class="group-header" ng-if="group && group.length > 0">
             <input
               type="text"
               ng-click="openFilters[filter.rangeUrlTemplate + 'Selector'] = !openFilters[filter.rangeUrlTemplate + 'Selector']"
-              value="{{filter.dateRangeLabel() | translate}}"
+              value="{{ filter.dateRangeLabel() | translate }}"
               readonly
-            >
+            />
```
Comment (ohmyguigs) [inline src/vtex-ng-filter.html]: from the above comment

File: src/vtex-ng-filter.html [hunk 4]
```diff
@@ -198,16 +205,13 @@ <h3 class="group-header" ng-if="group && group.length > 0">
...
-              >
+              <label class="checkbox" ng-if="filter.type == 'multiple'">
                 <input
                   type="checkbox"
-                  name="{{filter.name}}"
+                  name="{{ filter.name }}"
                   ng-model="item.selected"
                   ng-change="filter.updateSelectedCount()"
-                >
+                />
```
Comment (ohmyguigs) [inline src/vtex-ng-filter.html]: here also (should this really  be closing?)

File: src/vtex-ng-filter.html [hunk 5]
```diff
@@ -224,10 +228,10 @@ <h3 class="group-header" ng-if="group && group.length > 0">
               >
                 <input
                   type="radio"
-                  name="{{filter.name}}"
+                  name="{{ filter.name }}"
                   ng-model="filter.selectedItem"
                   ng-value="item"
-                >
+                />
```
Comment (ohmyguigs) [inline src/vtex-ng-filter.html]: lint pls (from the above comment)

File: src/vtex-ng-filter.html (hunk 2 above)
Comment (salesfelipe) [reply src/vtex-ng-filter.html]: dunno 🤷‍♂️

File: src/vtex-ng-filter.html [hunk 6]
```diff
@@ -13,6 +13,8 @@ <h3>
     </button>
   </h3>
 
+  <span ng-if="infoMessage" translate>{{ infoMessage }}</span>
```
Comment (arthurepc) [inline src/vtex-ng-filter.html]: do you need this translate attribute? Is `infoMessage` a string with the key to be translated or the string of the message already localized?

**Pull request merged**